client/src/main/java/org/apache/rocketmq/client/trace/AsyncTraceDispatcher.java

```diff
@@ -48,6 +35,14 @@
 import org.apache.rocketmq.logging.InternalLogger;
 import org.apache.rocketmq.remoting.RPCHook;
 
+import java.util.*;
```

Comment:
Don't "Import xxx.*'

Reply:
Sorry for the mistake. Might be autoformatted by the IDE. Already fixed it. Thx.